### ISSUE TITLE
broke custom mini plugin

### ISSUE THREAD

**Issue by manc0048** [LordAshes/TaleSpire-StatMessagingPlugin#3]
through R2modmanager i had this recently update and it then broke your custom mini plugin. Reverted this back to the previous version and it worked again in game

**Comment by LordAshes** [comment]
Please describe the issues that you were having when running CMP?

Was it related to the Radial UI menu? If so, that should be patched now.

Best Regards,

Lord Ashes

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by manc0048** [comment]
> 
Yes i believe it was. I would right click a mini to follow the standard procedure to transition it to a custom mini but after selecting this option from the radial ui no dialogue box would ever appear. 
P.S. thankyou for this mod, it's amazing :)

**Comment by manc0048** [comment]
Updated to your latest version, not working like stated at first. Here is the error from r2modmanager:

[Error  : Unity Log] MissingMethodException: void LordAshes.StatMessaging.Reset(string)
Stack trace:
LordAshes.CustomMiniPlugin.Update () (at <4a2187129edc4ccda43962133a688a8d>:0)

**Comment by LordAshes** [comment]
The latest StatMessaging (released today) should fix that issue.

Lord Ashes

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
